transport/server.hh

```diff
@@ -38,6 +38,7 @@
 #include <seastar/core/metrics_registration.hh>
 #include "utils/fragmented_temporary_buffer.hh"
 #include "service_permit.hh"
+#include "connection_notifier.hh"
 
```

Comment:
See https://godbolt.org/z/vqA968 for an example where forward declaration is sufficient.

Reply:
I'm retarded.